addons/mail/i18n/mail.pot

```diff
@@ -10901,4 +10910,4 @@ msgstr ""
 #: code:addons/mail/models/discuss/discuss_channel_member.py:0
 #, python-format
 msgid "“%(member_name)s” in “%(channel_name)s”"
-msgstr ""
+msgstr ""
```

Comment:
Missing newline at end of file.

Newlines are important; according to the UNIX Standard, a line is "a sequence of zero or more non-\<newline> characters plus a terminating \<newline> character". This means that without the newline at the end of the file, the last "line" is not considered a line in the UNIX sense. This can sometimes be a problem with some more pedantic programs: `wc`, for example, wouldn't count your last line.